packages/core/src/item.ts

```diff
@@ -193,6 +179,17 @@ export const FIELD_DEFS: { [t in FieldType]: FieldDef } = {
             return masked ? value.replace(/./g, "\u2022") : value;
         },
     },
+    [FieldType.Text]: {
+        type: FieldType.Text,
+        pattern: /.*/,
+        matchPattern: /.*/,
+        mask: false,
+        multiline: true,
+        icon: "text",
+        get name() {
+            return $l("Plain Text");
+        },
+    },
```

Comment:
Renamed the `text` field to "Plain Text" and enabled multi line editing. This basically replaces what the `note` field was previously.